### PR TITLE
Closes #2 Login user

### LINKED ISSUE #2: Registration and login
- add Passport.js auth with JWT
- add Users model to the DB
- add documentation for this API

### PR DESCRIPTION
**Pull request by oskvortsov-bmt**
Closes #2

### CODE REVIEW

File: api/controllers/v1/AuthController.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ *
+ * @description :: Server-side logic for managing Users
+ * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+ */
+
+
+const passport = require("passport");
+
+module.exports = {
+  login: login,
```
Comment (zaikin-andrew) [inline api/controllers/v1/AuthController.js]: ES6. http://es6-features.org/#PropertyShorthand

**Comment by oskvortsov-bmt** [comment]
via Hubstaff
User: Oleg Skvortsov

Project: Hirefy-API - https://app.hubstaff.com/projects/300614
Date Range: 10/20/17 - 10/20/17
Work session total: 0:54:09

Grand total: 0:54:09

File: config/policies.js [hunk 2]
```diff
@@ -19,6 +19,12 @@
 
 module.exports.policies = {
 
+  '*': ['isAuthenticated'],
```
Comment (zaikin-andrew) [inline config/policies.js]: probably for `v1/*`

File: package.json [hunk 3]
```diff
@@ -5,6 +5,8 @@
   "description": "a Sails application",
   "keywords": [],
   "dependencies": {
+    "bcrypt": "^1.0.3",
```
Comment (zaikin-andrew) [inline package.json]: remove

File: tasks/register/compileAssets.js [hunk 4]
```diff
@@ -17,6 +17,7 @@ module.exports = function(grunt) {
     'jst:dev',
     'less:dev',
     'copy:dev',
-    'coffee:dev'
+    'coffee:dev',
```
Comment (zaikin-andrew) [inline tasks/register/compileAssets.js]: Do we use `coffeescript`?

File: api/controllers/v1/UsersController.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  getUsers
+}
+
+function getUsers(req, res) {
+  Users.find().exec((err, usersNamedFind) => {
```
Comment (zaikin-andrew) [inline api/controllers/v1/UsersController.js]: what about hashed passwords? Do we return it?

File: api/controllers/v1/AuthController.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  if (!user) return res.unauthorized(null, info && info.code, info && info.message);
+
+  return res.ok({
+    // TODO: replace with new type of cipher service
+    token: `JWT ${CipherService.createToken(user)}`
+  });
+}
+
+/**
+ * @api {post} /v1/auth Sign in
```
Comment (zaikin-andrew) [inline api/controllers/v1/AuthController.js]: Add more info how to use token on the front end

**Pull request merged**